modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java

```diff
@@ -342,6 +343,17 @@ protected void validateOperationConfiguration(ConfigurationProvider configuratio
     }
   }
 
+  @Override
+  protected ParameterValueResolver getParameterValueResolver() {
+    return (fieldName) -> {
+      try {
+        return source.getFieldValue(fieldName);
```

Comment:
i'd prefer SourceWrapper to have a getDelegate() which returns te actual source, and then use instrospectionUtils inside the parameter resolver